policy_packs/aws/backup/enforce_ebs_volume_backups/policies.tf

```diff
@@ -0,0 +1,120 @@
+# AWS > Backup > Stack
+resource "turbot_policy_setting" "aws_backup_stack" {
+  resource = turbot_policy_pack.main.id
```

Comment:
Can you please fix alignment for all of these args, so they align with the farthest right `=`, like in https://github.com/turbot/guardrails-samples/blob/main/policy_packs/aws/ec2/enforce_approved_amis_publishers_for_instances/policies.tf#L2-L9